lab 2/index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <title>My title</title>
+
+      <!-- font awesome -->
+      <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+      <link rel="stylesheet" href="css/style.css">
+   </head>
+   <body>
+   <header>
+      <div class="collapsed-menu">
+         <div class="title menu-item">Navigation</div>
```

Comment:
if it's title, than it's better to use h1 h2 h3 h4 ...

Reply:
Well, it's not a title actually.
This class accidentily wasn't removed.
#done